### DISCUSSION TITLE
blog about deprecation

### DISCUSSION THREAD

**Discussion by Janatbek** [gulpjs/gulp#2508] [General] [answered]
since the medium is not free to read, and they hide content if not subscribed. deprecation notes I think should be in readme, or in release notes not in paid platforms like medium. I am referring to gulp-util is deprecated - replace it, following the guidelines at link https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5

**Comment by phated** [comment, marked as answer]
@Janatbek I'm not seeing any hidden content when logged out on Safari, but I know that Medium does some shady stuff. 

We'd be happy to move our blog over to our new website, but no one has taken on the task to figure out how to do that with Docusaurus v2 alpha. Is that something you'd want to take on? I can create an issue and assign it to you.
